components/ui/article-detail.tsx

```
@@ -7,8 +7,8 @@ interface Props {
 
 export default function ArticleDetail({ article }: Props) {
   return (
-    <article className="space-y-2">
-      <h2 className="text-xl font-bold">{article.title}</h2>
+    <article className="space-y-2 p-4 sm:p-6">
+      <h2 className="text-xl sm:text-2xl font-bold">{article.title}</h2>
       <div dangerouslySetInnerHTML={{ __html: article.content ?? '' }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)